app/design/adminhtml/default/default/template/zip/payment/system/config/field/check_credential_button.phtml

```diff
@@ -0,0 +1,82 @@
+<div>
+    <button id="zip-payment-credential-button" onclick="javascript:checkCredential();" class="scalable" type="button"
+    id="<?php echo $this->getHtmlId(); ?>">
+        <span>
+            <span>
+                <span >Check Zip credential</span>
+            </span>
+        </span>
+    </button>
+    <div id="zip-payment-healthcheck">
+    </div>
+</div>
+<style type="text/css">
+    #zip-payment-healthcheck span {
+        padding: 5px 10px;
+        text-transform: uppercase;
+        display: inline-block;
+        color: white !important;
+        font-weight: bold;
+        border-radius: .25rem;
+    }
+    #zip-payment-healthcheck span.success {
+        margin-top: 10px;
+        background-color: #28a745;
+    }
+    #zip-payment-healthcheck span.warning {
+        margin-top: 10px;
+        background-color: #ffc107;
+    }
+    #zip-payment-healthcheck span.error {
+        margin-top: 10px;
+        background-color: #bf0606;
+    }
+    #zip-payment-healthcheck li {
+        line-height: 1.2em;
+        border-bottom: dashed 1px #e3e3e3;
+        padding: 5px 0;
+        font-weight: 600 !important;
+    }
+    #zip-payment-healthcheck li.success {
+        color: #155724 !important;
+    }
+    #zip-payment-healthcheck li.warning {
+        color: #856404 !important;
+    }
+    #zip-payment-healthcheck li.error {
+        color: #bf0606 !important;
+    }
+</style>
+
+<script type="text/javascript">
+//<![CDATA[
+
+checkCredential = function() {
```

Comment:
Hi @mahbub-zip could you implement the similar function like Magento2 to POST the env, APIkey to the backend?